### ISSUE TITLE
[macOS] logging to file from command line does not work

### ISSUE THREAD

**Issue by gingerbeardman** [libretro/RetroArch#8135]
## Description

logging to file from command line does not work, RA just shows usage information.

### Expected behavior
should run RA whilst logging output to file

### Actual behavior

RA does not launch, quits back to command line with usage information

### Steps to reproduce the bug

1. cd to retroarch binary path
2. `./RetroArch --menu --verbose --log-file ~/Desktop/RetroArch.log`

### Bisect Results

Not tried logging in a long time, but it did work at one point

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 
  - 1.7.5 stable (22 Oct 2018)
  - latest nightly (31 Jan 2019)

### Environment information

- OS: macOS 10.13.6

**Comment by orbea** [comment]
Would you mind bisecting this? It does work for me with linux.

Do other forms of logging work on osx with redirection or even `tee(1)`?
```
retroarch --verbose > ~/Desktop/RetroArch.log
```
or
```
retroarch --verbose | tee ~/Desktop/RetroArch.log
```

**Comment by gingerbeardman** [comment]
Both of those work, but only show INFO logging.

Same if I just run `./RetroArch --verbose` or even `./RetroArch` on the command line.

But my RA is set to DEBUG logging so should show way more, right?

**Comment by gingerbeardman** [comment]
They both launch RA but no logging is in the file.

Also, perhaps separate issue, I am seeing only INFO output when my RA logging level is set to DEBUG so I should see way more?

![screen shot 2019-01-31 at 16 54 06](https://user-images.githubusercontent.com/49612/52070506-e03e6700-2578-11e9-838a-ede7404f81c2.png)

**Comment by gingerbeardman** [comment]
OK, sorry for editing my previous post.

`./RetroArch --verbose > ~/Desktop/RetroArch.log`
and
`./RetroArch --verbose | tee ~/Desktop/RetroArch.log`

both give a log file with:
```
Memcard 2: disabled
Starting PCSX-ReARMed 0
Running PCSX Version 1.9 (Oct 12 2018).
psxMap: warning: wanted to map @80000000, got 0x11309d000
psxMap: warning: wanted to map @77000000, got 0x11309d000
psxMap: warning: wanted to map @14000000, got 0x119473000
psxMap: warning: wanted to map @1f800000, got 0x11309d000
psxMap: warning: wanted to map @14000000, got 0x115a22000
psxMap: warning: wanted to map @1fc00000, got 0x11309d000
psxMap: warning: wanted to map @14000000, got 0x119690000
Mapped (RAM/scrp/ROM/LUTs/TC):
19480000/15a30000/19690000/00000000/00000000
found BIOS file: SCPH5501.bin
RGB565 supported, using it
plugin: plugins/builtin_gpu
plugin: plugins/builtin_spu
plugin: plugins/builtin_pad
plugin: plugins/builtin_pad
Plugins loaded.
Loaded CD Image: /Users/matt/Downloads/2019-01-31/The Hanafuda 10.PBP[pbp].
Track 01 (DATA) - Start 00:02:00, Length 00:32:05
Track 02 (AUDIO) - Start 00:34:05, Length 02:04:68
Track 03 (AUDIO) - Start 02:38:73, Length 02:23:56
Track 04 (AUDIO) - Start 05:02:54, Length 02:54:17
Track 05 (AUDIO) - Start 07:56:71, Length 02:44:14
Track 06 (AUDIO) - Start 10:41:10, Length 02:36:20
selected sound output driver: libretro
CD-ROM Label: SLPS_01684                      
CD-ROM ID: SLPS01684
CD-ROM EXE Name: SLPS_016.84;1
ari64_reset
```

**Comment by gingerbeardman** [comment]
Beetle does not produce any logging at all.

**Comment by gingerbeardman** [comment]
This is fixed as of 1.7.7, not sure when exactly it was fixed.
